src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -178,6 +211,78 @@ function DatasetOverviewPanel(props) {
             <Typography variant="h6">Cloud Platforms:</Typography>
             {renderCloudPlatforms(dataset)}
           </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Dataset Id:</Typography>
+            <Typography>{dataset.id}</Typography>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Default Billing Profile Id:</Typography>
+            {dataset.defaultProfileId}
+          </Grid>
+          {dataset.dataProject && (
+            <Grid item xs={4}>
+              <Typography variant="h6">Google Data Project:</Typography>
+              {dataset.dataProject}
+            </Grid>
+          )}
+          {dataset.ingestServiceAccount && (
+            <Grid item xs={4}>
+              <Typography variant="h6">Ingest Service Account:</Typography>
+              <Typography>{dataset.ingestServiceAccount}</Typography>
+            </Grid>
+          )}
+          <Grid item xs={4}>
+            <Typography variant="h6">Secure Monitoring Enabled?</Typography>
+            {dataset.secureMonitoringEnabled ? 'Yes' : 'No'}
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Self Hosted?</Typography>
+            {dataset.selfHosted ? 'Yes' : 'No'}
+          </Grid>
+          <Grid item xs={4}>
+            <EditableFieldView
+              fieldValue={dataset.phsId}
+              fieldName="PHS Id"
+              canEdit={userRoles.includes(DatasetRoles.STEWARD)}
+              updateFieldValueFn={(text) => dispatch(patchDataset(dataset.id, { phsId: text }))}
+              useMarkdown={false}
+            />
+          </Grid>
+          {dataset.schema.relationships.length > 0 && (
```

Comment:
Do you have a dataset with relationships that I can peek at? Thank you!

Reply:
Yes - should be an example here: https://jade-sh.datarepo-dev.broadinstitute.org/datasets/c5248002-05d6-469e-bf31-38f1792198f3